client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -713,6 +713,26 @@ Undocumented.prototype.metaKeyring = function( fn ) {
 };
 
 /**
+ * Upload a Gravatar image
+ *
+ * @param {string} email The email address of the Gravatar account
+ * @param {Blob} file The file data blob of the image
+ * @param {Function} fn The callback function
+ * @return {Promise} A Promise to resolve when complete
+ * @api public
+ */
+Undocumented.prototype.uploadGravatar = function( email, file, fn ) {
```

Comment:
It's just the pattern in all the functions in `wpcom`. If a `fn` callback is provided, it will use the callback; otherwise it will return a Promise. tbh, I'd rather just skip the callback and rely on the promise instead.